src/test/java/lotto/step3/domain/WinningNumbersTest.java

```diff
@@ -0,0 +1,22 @@
+package lotto.step3.domain;
+
+import lotto.step3.enums.MatchNumber;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static lotto.step3.domain.Fixture.lottosList;
+import static lotto.step3.domain.Fixture.winningNumber;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class WinningNumbersTest {
+    @Test
+    @DisplayName("기존 5개 번호가 일치하고 나머지 하나의 번호가 보너스 번호와 일치하면 MatchNumber-SECOND 반환")
+    void bonusNumber() {
+        WinningNumbers winningNumbers = new WinningNumbers(winningNumber);
+        Lottos lottos = new Lottos(lottosList);
+        BonusNumber bonusNumber = new BonusNumber(Fixture.bonusNumber, winningNumbers);
+        Result result = new Result(winningNumbers.getWinnerStat(lottos, bonusNumber.getBonusNumber()));
+        result.getStat().forEach((k, v) -> assertThat(k).isEqualTo(MatchNumber.SECOND));
```

Comment:
```suggestion
        Result result = 당첨번호.rank(당첨번호와_한자리가_다르며_보너스번호를_가지는_로또번호, 보너스번호);
        assertThat(result.contains(Rank.SECOND)).isTrue();
```


- 주입되는 값들이 범용적으로 사용되는 것이 아닌데요. 이런 경우는 지역변수로 위치하는게 가독성에 좋다고 생각해요.
- 혹은 값을 미리 구성해두고 싶다면, 변수명을 위와 같이 바로 식별가능하도록 작성하는 것이 좋다고 생각해요.

Reply:
앗 감사합니다.
한글 변수가 가독성이 좋네요!